aviary/examples/external_subsystems/OAS_weight/OAS_wing_weight_analysis.py

```diff
@@ -220,30 +231,24 @@ def compute(self, inputs, outputs):
         surf_dict = {
             # surface name
             'name': 'meshwing',
-
             # wing symmetry
             'symmetry': self.options['symmetry'],
-
             # wing mesh
             'mesh': mesh,
-
             # wing definition
             'S_ref_type': self.options['S_ref_type'],
             'fem_model_type': self.options['fem_model_type'],
-
             # wing thickness data
             'data_x_upper': inputs['box_upper_x'],
             'data_x_lower': inputs['box_lower_x'],
             'data_y_upper': inputs['box_upper_y'],
             'data_y_lower': inputs['box_lower_y'],
-
             # wing sizing parameters
             'twist_cp': inputs['twist_cp'],
             'spar_thickness_cp': inputs['spar_thickness_cp'],
             'skin_thickness_cp': inputs['skin_thickness_cp'],
             't_over_c_cp': inputs['t_over_c_cp'],
             'original_wingbox_airfoil_t_over_c': inputs['airfoil_t_over_c'],
-
```

Comment:
Wait, why is Ruff deleting every blank line? That makes code harder to read, and is a deal-breaker for me.

Reply:
ok, this is inside of a dictionary, so it doesn't happen in code. Still, maybe we should disable the auto-formatter in some of these.